src/main/java/pl/javastart/task/Group.java

```diff
@@ -0,0 +1,55 @@
+package pl.javastart.task;
+
+public class Group {
+
+    private String groupId;
+    private String groupName;
+    private int lecturerId;
+    private int[] studentIndex;
```

Comment:
Zamiast int[] lepiej byłoby przechowywać tutaj referencje, czyli Student[]. Znacznie ułatwiłoby to później wyświetlanie w metodzie printGroupInfo z UniversityApp